src/features/auth/pages/Login.test.tsx

```diff
@@ -1,23 +1,26 @@
 import { NavigationContainer } from '@react-navigation/native' // @react-navigation
 import { createStackNavigator } from '@react-navigation/stack' // @react-navigation
 import { render, fireEvent, waitFor, act } from '@testing-library/react-native'
+import { rest } from 'msw'
 import React from 'react'
-import { Text } from 'react-native'
 
-import { NotAuthenticatedError } from 'libs/fetch'
-import { IUser } from 'types'
-
-import * as api from '../api'
+import { Home } from 'features/home/pages/Home'
+import { server } from 'testUtils/server'
```

Comment:
```suggestion
import { server } from 'jest/server'
```
Si tu arrives à faire un alias sur le dossier jest, encore mieux.